Documentation/maintaining-debian-packages.md

```diff
@@ -90,6 +90,48 @@ history into Debian patches. If the patches are modified, commit the changes.
 
 To start from stratch, use `gbp pq drop`.
 
+### Typical session with patches
+
+A typical session with gbp can look as follows. Lets assume you have the code and
+you are in the repo root directory. Furthermore, lets assume that you want to 
+change something and rebuild and that you have just cloned (the tree is clean).
+
+1. run:
+
+```bash
+gbp pq import
+```
+It will switch to `patch-queue/debian/sid` branch.
+
+2. do make your changes
+
+3. commit them
+
+4. run:
+
+```bash
+gbp pq export
+```
+
+It will will export all the patches to `debian/patches` where you can see them.
+
+5. run:
+
+```bash
+gbp buildpackage --git-pbuilder --git-pbuilder-options=--source-only-changes -us -uc --git-ignore-new
+```
+
+to rebuild.
+
+6. when you want to add more changes, run:
+
+```bash
+git checkout patch-queue/debian/sid
```

Comment:
To do it the Debian way, this should be `gbp pq switch`.

It would also be nice to document that if you want to start from scratch, you can do `gbp pq drop` and then `gbp pq import`

Reply:
sure.